src/Tmds.ExecFunction/Tmds.ExecFunction.csproj

```diff
@@ -13,4 +13,23 @@
     <Copyright>Tom Deseyn</Copyright>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;NU5100</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\VsDebugger\VsDebugger.csproj">
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
+      <Private>true</Private>

```

Comment:
This `Private` means that a project referencing `Tmds.ExecFunction` doesn't include the `VsDebugger` executable, right?

Can you add a comment in the XML file about this.